examples/astro/src/env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="astro/client" />
```

Comment:
looks like this file got created somehow when i ran `pnpm run test` or `pnpm run lint` or `pnpm run typecheck`.

should it be added to .gitignore maybe?

Reply:
Ye probably - I've not Skipper checking it in 😅